lambdas/services/create_document_reference_service.py

```diff
@@ -132,7 +121,7 @@ def prepare_doc_object(
             reference_id=s3_object_key,
             content_type=validated_doc.contentType,
             file_name=validated_doc.fileName,
-            doc_type=document_type,
+            doc_type=validated_doc.docType,
```

Comment:
doc_type should be document_type, however, i think this was pre existing and links to dynamo so is no longer changeable.